server/controllers/article.controller.js

```diff
@@ -25,13 +30,15 @@ class ArticleController {
       title, body, tagList, image, description, published,
     } = req.body;
     const { id } = req.user;
+    const readingTime = await readTime(body);
```

Comment:
@geekdanitek are you only catering for the body of an article, what then happens to images?

Reply:
Currently, we are catering for only the article body